src/resources/favorites.ts

```diff
@@ -13,78 +13,9 @@ import { externalTokenQueryKey, fetchExternalToken } from './assets/externalAsse
 
 export const favoritesQueryKey = createQueryKey('favorites', {}, { persisterVersion: 2 });
 
-const getUniqueId = (address: AddressOrEth, chainId: ChainId) => getStandardizedUniqueIdWorklet({ address, chainId });
+const DEFAULT_FAVORITES = [DAI_ADDRESS, ETH_ADDRESS, SOCKS_ADDRESS, WBTC_ADDRESS];
```

Comment:
replaced the predefined DEFAULT map, with lacking info, for prefetching the default tokens, only once, all subsequent app loads will grab from the persisted cache